acra/configuring-maintaining/logging/output.md

```diff
@@ -0,0 +1,60 @@
+---
+weight: 1
+title: Logging output
+bookCollapseSection: true
+---
+
+# Logging output
```

Comment:
This is more like a list of supported logging formats, so I believe it should be titled like that, on a page named `/logging/formats/`